Plutarch/Builtin.hs

```diff
@@ -71,7 +78,7 @@ pasByteStr :: Term s (PData :--> PByteString)
 pasByteStr = punsafeBuiltin PLC.UnBData
 
 pdataLiteral :: Data -> Term s PData
-pdataLiteral = punsafeConstant . PLC.Some . PLC.ValueOf PLC.DefaultUniData
+pdataLiteral = pconstant
```

Comment:
Should we retain functions like `pdataLiteral`, or remove them while replacing their calls with `pconstant`?

Reply:
We should deprecate the old ones.

Reply:
Done.